docs/management/advanced-options.asciidoc

```diff
@@ -74,3 +76,14 @@ Markdown.
 `notifications:lifetime:error`:: Specifies the duration in milliseconds for error notification displays. The default value is 300000. Set this field to `Infinity` to disable error notifications.
 `notifications:lifetime:warning`:: Specifies the duration in milliseconds for warning notification displays. The default value is 10000. Set this field to `Infinity` to disable warning notifications.
 `notifications:lifetime:info`:: Specifies the duration in milliseconds for information notification displays. The default value is 5000. Set this field to `Infinity` to disable information notifications.
+
+`timelion:showTutorial`:: When enabled, the Timelion tutorial will be shown when a user first opens Timelion.
```

Comment:
I assume this is a boolean setting. If so, consider wording it like other boolean settings, something like

> `timelion:showTutorial`:: Set this property to `true` to show the Timelion tutorial to users when they first open Timelion.